examples/integration/iOSApp/Source/BUILD

```diff
@@ -44,7 +44,6 @@ ios_application(
         ":release_build": [],
         "//conditions:default": [
             "--digest-algorithm=sha1",
-            "--resource-rules=$(RESOURCE_RULES)",
```

Comment:
This was failing in Bazel HEAD job because `RESOURCE_RULES` is no longer defined